roles/pid_generator/files/gen_pids.py

```diff
@@ -245,8 +246,7 @@ def openLocalPidDB():
 except Exception as e:
     if DEBUG:
         logging.exception("Exception found. Script haltet.")
-	raise # re-raise the exception                                          
-	      # traceback gets printed                                          
```

Comment:
Why we no longer want this?

Reply:
I think I was overzealous there, re-added code.